scripts/towncrier-draft-to-file.py

```diff
@@ -0,0 +1,14 @@
+from subprocess import check_call
+
+
+def main():
+    """
+    Platform agnostic wrapper script for towncrier.
+    Fixes the issue (#7251) where windows users are unable to natively run tox -e docs to build pytest docs.
+    """
+    with open("doc/en/_changelog_towncrier_draft.rst", "w+") as f:
+        check_call(("towncrier", "--draft"), stdout=f)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
a better output (when this fails) is going to be:

```python
def main():
    with ...:
        return subprocess.call(...)

if __name__ == '__main__':
    exit(main())
```

that way the script exits nonzero but you don't see the `CalledProcessError` stacktrace